src/selector/var/whitespace.js

```diff
@@ -0,0 +1,6 @@
+define( function() {
+	"use strict";
+
+	// http://www.w3.org/TR/css3-selectors/#whitespace
```

Comment:
Might as well make this HTTPS now.

Reply:
Nice catch. I fixed the URL in the other PR but since this declaration has moved I could have missed that URL.